.github/CHANGELOG.md

```diff
@@ -14,14 +14,17 @@
   circuit layout. Without a layout, the gate parameters cannot be validated against the device
   specification.
   [(#661)](https://github.com/XanaduAI/strawberryfields/pull/661)
+  
+* The teleportation tutorial `examples/teleportation.py` now uses the correct value (now ϕ=0 instead of ϕ=π/2) for the phase shift of the beam     splitters.
+  [(#674)](https://github.com/XanaduAI/strawberryfields/pull/674)
```

Comment:
Although I really like the look of the ϕ and the π, I'd say it's better to go with the names used in the function, programmatically, here instead (`phi` and `np.pi`).
```suggestion
* The teleportation tutorial `examples/teleportation.py` now uses the correct value (now `phi = 0` instead of `phi = np.pi / 2`) for the phase shift of the beamsplitters.
  [(#674)](https://github.com/XanaduAI/strawberryfields/pull/674)
```